cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -676,12 +651,10 @@ script "js_instances_costs_request", type:"javascript" do
     day = day.length > 1 ? day : '0' + day
     return year + '-' + month + '-' + day
   }
-
   start_date = getFormattedDailyDate(new Date(new Date().setDate(new Date().getDate() - 3)))
   end_date = getFormattedDailyDate(new Date(new Date().setDate(new Date().getDate() - 2)))
-
   result = [{
-    "dimensions": ["resource_id"],
+    "dimensions": ["resource_id", "vender_account_name"],
```

Comment:
`"vender_account_name"` is this possibly a typo?

Reply:
vendor_account_name is to pull the display name of the vendor account from optima

Reply:
apologies, I meant it says 'vender' rather than 'vendor'

Reply:
ohhh good catch. thank you!